src/tasks/dump.ts

```diff
@@ -460,15 +460,56 @@ async function createOrders(
   receiver: Receiver,
   domainSeparator: TypedDataDomain,
   validTo: number,
+  maxFeePercent: number,
   api: Api,
 ) {
   for (const inst of instructions) {
+    const sellToken = inst.token.address;
+    const buyToken = isNativeToken(toToken) ? BUY_ETH_ADDRESS : toToken.address;
+    let feeAmount, sellAmount, buyAmount;
+    try {
+      // Re-quote for up-to-date fee (in case approval took long)
+      const updatedQuote = await api.getQuote({
+        sellToken,
+        buyToken,
+        sellAmountBeforeFee: inst.balance,
+        kind: OrderKind.SELL,
+        appData: APP_DATA,
+        partiallyFillable: false,
+        validTo,
+        from: await signer.getAddress(),
```

Comment:
Nit: `signer` is of type `SignerWithAddress` but at some point in the function stack it was changed to `Signer`. If you change the type to `SignerWithAddress` you should be able to do `signer.address`.